yarn.lock

```diff
@@ -216,16 +216,6 @@
     enabled "2.0.x"
     kuler "^2.0.0"
 
-"@esbuild/android-arm@0.15.12":
-  version "0.15.12"
-  resolved "https://registry.yarnpkg.com/@esbuild/android-arm/-/android-arm-0.15.12.tgz#e548b10a5e55b9e10537a049ebf0bc72c453b769"
-  integrity sha512-IC7TqIqiyE0MmvAhWkl/8AEzpOtbhRNDo7aph47We1NbE5w2bt/Q+giAhe0YYeVpYnIhGMcuZY92qDK6dQauvA==
-
-"@esbuild/linux-loong64@0.15.12":
-  version "0.15.12"
```

Comment:
And also revert it

Reply:
i have not edited anything, so i am not sure there is anything to revert.

Reply:
This file has changed since you upgraded `firebase-tools`.

Reply:
yes i figured that out. Do you have any idea how to rectify this?

Reply:
I have never tried this method but you try it maybe it will work. [Git Revert FIle](https://dev.to/lofiandcode/git-and-github-how-to-revert-a-single-file-dha)

Reply:
Ok, thank you! I will look into this.